x/auth/keeper/keeper.go

```diff
@@ -99,9 +99,9 @@ type AccountKeeper struct {
 	authority string
 
 	// State
-	Schema collections.Schema
-	Params collections.Item[types.Params]
-
+	Schema        collections.Schema
+	Params        collections.Item[types.Params]
+	AccountNumber collections.Sequence
```

Comment:
Personal preference: would it make sense to make this private and provide a deprecated method to peek the value in the upgrade handler? I think we should ensure that this not used by other modules anymore and remove it in a future release.

Reply:
yeah good idea